README.md

```diff
@@ -60,6 +60,6 @@ jobs:
       - name: validate translations
         run: |
           swift build --package-path ../SwiftPolyglot --configuration release
-          swift run --package-path ../SwiftPolyglot swiftpolyglot "es,fr,de,it" --errorOnMissing
+          swift run --package-path ../SwiftPolyglot swiftpolyglot es fr de it --errorOnMissing
```

Comment:
I think the flag needs changing to `--error-on-missing` here

Reply:
Nice catch! Fixed!